packages/cozy-mespapiers-lib/src/components/Papers/Empty/Empty.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import UiEmpty from 'cozy-ui/transpiled/react/Empty'
+
+import HomeCloud from '../../../assets/icons/HomeCloud.svg'
+import EmptyWithConnector from './EmptyWithConnector'
+
+const Empty = ({ connector, accounts }) => {
```

Comment:
it seems this component can result in multiple different views depending on conditions, maybe we can do some integration testing on this one to check the scenarios

Reply:
je rajoute un commit à la fin